src/System.Diagnostics.FileVersionInfo/src/System.Diagnostics.FileVersionInfo.builds

```diff
@@ -9,6 +9,11 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Diagnostics.FileVersionInfo.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Yeah, I first made the unix one AnyOS and TargetGroup='' but then I had to use PackageDestination to copy the build to runtimes/win/lib/netcore50 as well as lib/netstandard.  That felt less clean than this.